xfce4-desktop/README.md

```diff
@@ -0,0 +1,49 @@
+# XFCE4 Desktop
+
+An XFCE4 desktop environment presented via TurboVNC and a novnc proxy.
+
+## Running the Workflow
```

Comment:
As a prerequisite to running the workflow, a user should have a Mountain access key to use to the container in Mountain. I would recommend referencing https://beta.fuzzball.io/docs/appendices/cli-installation/#signing-into-mountain-and-creating-a-key